s3/s3_transport/unit_tests/src/test_s3_transport.cpp

```diff
@@ -18,8 +18,8 @@
 #include <sstream>
 #include <string_view>
 #include <fmt/format.h>
-#include "irods/miscServerFunct.hpp"
-#include "irods/data_object_finalize.h"
+#include <irods/miscServerFunct.hpp>
+//#include <irods/data_object_finalize.h>
```

Comment:
If it's not needed, we should just delete it.

Reply:
yes. please remove that line and squash it.

once that is done, i think we're good.

Reply:
done